.npmignore

```diff
@@ -1 +1,10 @@
-logo
+.editorconfig
+.eslintcache
+.eslintignore
+.eslintrc.json
+.github
+.lintstagedrc
+assets
```

Comment:
No idea what you'd call them otherwise. Maybe they should just go in the documentation folder as they're only used there.